### PR TITLE
Add Joplin

### PR DESCRIPTION
**Pull request by renaudjx**
Thank you for taking the time to work on a PR for Awesome-Selfhosted!

To ensure your PR is dealt with swiftly please check the following:

- [x] Your submissions are formatted according to the following requirements:
        
``- [Name](http://homepage/) - Short description, less than 250 characters. ([Demo](http://url.to/demo), [Source Code](http://url.of/source/code), [Clients](https://url.to/list/of/related/clients-or-apps)) `License` `Language` ``

Depends on proprietary services:

``- [Name](http://homepage/) `⚠` - Short description, less than 250 characters. ([Demo](http://url.to/demo), [Source Code](http://url.of/source/code), [Clients](https://url.to/list/of/related/clients-or-apps)) `License` `Language` ``

Non-free software in `non-free.md`:

``- [Name](http://homepage/) `⊘ Proprietary` - Short description, less than 250 characters. ([Demo](http://url.to/demo), [Source Code](http://url.of/source/code), [Clients](https://url.to/list/of/related/clients-or-apps)) `Language` ``

- [x] Your additions are ordered alphabetically.
- [x] Your additions are [Free software](https://en.wikipedia.org/wiki/Free_software), or if not they have been added to [non-free](non-free.md) and marked `⊘ Proprietary`.
- [x] If your additions depend on proprietary services outside the user's control, they must be marked `⚠`.
- [x] Your additions are not already listed at [awesome-sysadmin](https://github.com/n1trux/awesome-sysadmin) (IT infrastructure management), [staticgen.com](https://www.staticgen.com/) or [staticsitegenerators.net](https://staticsitegenerators.net/) (static site generators).
- [x] Your additions have their `License` main server-side `Language`/platform/requirement listed.
- [x] Any licenses you have added are in our [list of licenses](https://github.com/Kickball/awesome-selfhosted/blob/master/README.md#list-of-licenses).
- [x] You have searched the repository for any relevant [issues](https://github.com/Kickball/awesome-selfhosted/issues) or [PRs](https://github.com/Kickball/awesome-selfhosted/pulls), incluing closed ones.
- [x] If the main link of your addition points to the program's source code, there is no need to add a duplicate _Source Code_ link. The _Demo_ and _Clients_ links are not mandatory.
- [x] Any category you are creating has the minimum requirement of 3 items. If not, your addition may be inserted into `Misc/Other`.
- [x] Any software project you are adding to the list is actively maintained.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -958,6 +958,7 @@ See https://github.com/n1trux/awesome-sysadmin#monitoring and https://github.com
 - [Standard Notes](https://standardnotes.org) - Simple and private notes app. Protect your privacy while getting more done. That's Standard Notes. ([Demo](https://app.standardnotes.org/), [Source Code](https://github.com/standardnotes)) `GPL-3.0` `Ruby`
 - [to-markdown](https://domchristie.github.io/to-markdown/) - HTML to Markdown converter written in JavaScript. ([Source Code](https://github.com/domchristie/to-markdown)) `MIT` `JavaScript`
 - [Turtl](https://turtl.it/) - Totally private personal database and note taking app. ([Source Code](https://github.com/turtl)) `GPL-3.0` `CommonLisp`
+- [Joplin](https://joplin.cozic.net/) - An open source note taking and to-do application with synchronisation capabilities. ([Source Code](https://github.com/laurent22/joplin)) `MIT` `JavaScript`
```
Comment (Kickball) [inline README.md]: Please order the entry alphabetically based on project name.

Comment (Kickball) [inline README.md]: Does the application operate a server-client setup or is it a desktop only client based?

The latter would make the Joplin ineligible for this list.

**Comment by Kickball** [comment]
@renaudjx thank you for putting the time into making this request, I have made a few comments above. Let me know if anything is unclear.

**Comment by nodiscc** [comment]
From the website homepage:

>Applications
>    Desktop application
>    Mobile applications
>    Terminal application
>    Web Clipper

So it looks like this is purely a desktop/mobile application, not server-side software. I don't think it qualifies for the list. Please reopen if you think otherwise.

**Pull request closed without merging**